productivity-suite/app/fragments/todo-lists/src/dispatchSelectedListUpdated.ts

```diff
@@ -1,16 +1,11 @@
-import { dispatchPiercingEvent } from "piercing-library";
+import { getBus } from "piercing-library";
 import type { TodoList } from "shared";
 
 export function dispatchSelectedListUpdated(
   el: Element,
-  listSelected: TodoList,
-  which?: "previous" | "next"
```

Comment:
I've totally remove the animation logic for now, I am considering if we do want it or not (since now the todos fragment doesn't get removed and added, so it looks nice even without any entering/leaving animation)